README.md

```diff
@@ -60,7 +61,13 @@ by Neutrino. This means that by default all project source code should live in a
 project. This includes JavaScript files, CSS stylesheets, images, and any other assets that would be available
 to import your compiled project.
 
-## Quickstart
+| Project Type | Source | Output |
+| --- | --- | --- |
+| React app | `src` | `build` |
+| Web Library | `src` | `lib` |
+| React components | `src/components` | `lib` |
+
+## Quickstart React App
 
 After installing Neutrino and the this preset, add a new directory named `src` in the root of the project, with
 a single JS file named `index.js` in it.
```

Comment:
Since we're dealing with a React app, perhaps it would be useful to mention to have `react` and `react-dom` installed.